resources/js/components/open/forms/components/form-components/FormAboutSubmission.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <collapse class="p-5 w-full" :default-value="true">
+  <collapse class="py-3 w-full border-b" :default-value="true">
     <template #title>
       <h3 class="font-semibold text-lg relative">
-        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-gray-500 mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5 inline text-nt-blue mr-2 -mt-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Comment:
no `nt-blue`   
`nt` is for "notion-tools", the old Name of NotionForms

Reply:
we still not removed from class name so it's working

Reply:
Yes but let's not use anymore, anyway this colors isn't used in figma